src/plugin/relativeTime/index.js

```diff
@@ -44,10 +84,10 @@ export default (o, c, d) => {
           ? d(input).diff(instance, t.d, true)
           : instance.diff(input, t.d, true)
       }
-      const abs = Math.round(Math.abs(result))
+      const abs = Math[o.strict ? 'floor' : 'round'](Math.abs(result))
```

Comment:
Another difference between two modes is here, `0.1s` would display `0 second` in `strict` mode.

Reply:
we can leave this to Additional token processing https://day.js.org/docs/en/customization/relative-time#additional-token-processing

Reply:
Then we have to write a few methods for `ss`, `mm`, `hh`, `dd`, etc.?
That seems terrible to me.